assets/release/Dockerfiles/Dockerfile.ubuntu-22.04

```diff
@@ -26,6 +28,7 @@ ENV PATH $CARGO_HOME/bin:$PATH
 RUN mkdir -p "$CARGO_HOME" && mkdir -p "$RUSTUP_HOME" && \
     curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable && \
     chmod -R a=rwX $CARGO_HOME $RUSTUP_HOME
+RUN rustup target add wasm32-wasi && rustup target add wasm32-unknown-unknown
```

Comment:
And another minor thing: can we move this to the below `RUN` step? This `RUN` step is the same for all release images; only this image has an additional `RUN` step to install CI/CD dependencies below after the "Note" comment. We can also avoid a new  `RUN` step there and do `&& \ [newline] rustup target add ...` anywhere you see fit in that chain of commands down there.

Also since the Rustup target install step in the CI is skipped on Act, it makes me wonder how come Act worked then? 🤔

Reply:
Thank you for noticing that! I will accommodate `rustup` in that last chain of commands. 

> Also since the Rustup target install step in the CI is skipped on Act, it makes me wonder how come Act worked then?

That also stroke me as odd. In a recent [commit](https://github.com/Kong/ngx_wasm_module/commit/70552dd1ac7af07fb13e70213ab2d2dc84451b25#diff-eb2b51a21328577173b0f626df0ca814350888705a59f9575e08538acf532fcf) we introduced `if: ${{ !env.ACT }}` to the steps installing rust and go in both `unit` and `valgrind` jobs. Although the change made sense as both dependencies are installed in the docker image used by act, it lacked rust targets setup.

Act was consistently failing jobs due to missing rust targets:
```
| error[E0463]: can't find crate for `core`
|   |
|   = note: the `wasm32-wasi` target may not be installed
|   = help: consider downloading the target with `rustup target add wasm32-wasi`
```

Then after including the rustup commands in the Dockerfile, jobs successfully compiled rust plugins.